### PR TITLE
Move address converter to a separate page in Tools

### PR DESCRIPTION
**Pull request by fadeev**
- Checks checksum
- Button for copying
- QR code

Screenshot 2024-09-11 at 17 37 29

## Summary by CodeRabbit

- **New Features**
	- Enhanced the AddressConverter component for improved validation and conversion of cryptocurrency addresses.
	- Introduced new packages for QR code generation capabilities.

- **Documentation**
	- Updated documentation to streamline user guidance for address conversion, now linking to an external reference.
	- Added new documentation for the address conversion tool, detailing its functionality and usage.

- **Bug Fixes**
	- Improved error handling and user feedback for address input in the AddressConverter component.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,9 +62,11 @@
     "nextra": "2.13.4",
     "nextra-theme-docs": "2.13.4",
     "openai": "4.52.7",
+    "qrcode.react": "^4.0.1",
```
Comment (GMaiolo) [inline package.json:65]: This is unused

File: src/components/Docs/components/AddressConverter/components/AddressConverter.tsx [hunk 2]
```diff
@@ -1,40 +1,163 @@
...
+          <button
+            onClick={copyToClipboard}
+            className="border border-gray-200 dark:border-gray-500 rounded p-3 hover:border-green-100 hover:dark:border-green-100 transition min-w-[150px] whitespace-nowrap"
+          >
+            Copy
+          </button>
+        </div>
+      )}
+
+      {convertedAddress && (
```
Comment (GMaiolo) [inline src/components/Docs/components/AddressConverter/components/AddressConverter.tsx:154]: This is the same check as above, can be in within the same check, maybe wrapping it within a fragment?

**Review by GMaiolo — APPROVED** [review]
utACK
Left a couple of minor comments

File: package.json (hunk 1 above)
Comment (fadeev) [reply package.json]: 4235b85

File: src/components/Docs/components/AddressConverter/components/AddressConverter.tsx [hunk 3]
```diff
@@ -1,40 +1,163 @@
...
+    } catch (error) {
+      setErrorMessage("Conversion failed, but the address is valid.");
+      console.error("Conversion error:", error);
+    }
+  };
+
+  const copyToClipboard = () => {
+    if (convertedAddress) {
+      navigator.clipboard.writeText(convertedAddress);
+      alert("Address copied to clipboard!");
```
[the remark this answers is not in the document]
Comment (fadeev) [reply src/components/Docs/components/AddressConverter/components/AddressConverter.tsx]: 4c8c2be

File: src/components/Docs/components/AddressConverter/components/AddressConverter.tsx (hunk 2 above)
Comment (fadeev) [reply src/components/Docs/components/AddressConverter/components/AddressConverter.tsx]: 69cbbb6

**Pull request merged**